docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+
+- **Default**: 60000
+- **Unit**: N/A
+- **Description**: The minimum number of file descriptors in the BE process.
+
+#### index_stream_cache_capacity
+
+- **Default**: 10737418240
+- **Unit**: Byte
+- **Description**: The cache capacity for the statistical information of BloomFilter, Min, and Max.
+
+#### disable_storage_page_cache
+
+- **Default**: FALSE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to disable PageCache.
+  - When PageCache is enabled, StarRocks caches the recently scanned data.
+  - PageCache can significantly improve the query performance when similar queries are repeated frequently.
+  - TRUE indicates disabling PageCache.
+  - The default value of this item has been changed from TRUE to FALSE since StarRocks v2.4.
+
+#### base_compaction_num_threads_per_disk
+
+- **Default**: 1
+- **Unit**: N/A
+- **Description**: The number of threads used for Base Compaction on each storage volume.
+
+#### base_cumulative_delta_ratio
+
+- **Default**: 0.3
+- **Unit**: N/A
+- **Description**: The ratio of cumulative file size to base file size. The ratio reaching this value is one of the conditions that trigger the Base Compaction.
+
+#### compaction_trace_threshold
+
+- **Default**: 60
+- **Unit**: Second
+- **Description**: The time threshold for each compaction. If a compaction takes more time than the time threshold, StarRocks prints the corresponding trace.
+
+#### be_http_port
+
+- **Default**: 8040
+- **Unit**: N/A
+- **Description**: The HTTP server port.
+
+#### be_http_num_workers
+
+- **Default**: 48
+- **Unit**: N/A
+- **Description**: The number of threads used by the HTTP server.
+
+#### load_data_reserve_hours
+
+- **Default**: 4
+- **Unit**: Hour
+- **Description**: The reservation time for the files produced by small-scale loadings.
+
+#### number_tablet_writer_threads
+
+- **Default**: 16
+- **Unit**: N/A
+- **Description**: The number of threads used for Stream Load.
+
+#### streaming_load_rpc_max_alive_time_sec
+
+- **Default**: 1200
+- **Unit**: Second
+- **Description**: The RPC timeout for Stream Load.
+
+#### fragment_pool_thread_num_min
+
+- **Default**: 64
+- **Unit**: N/A
+- **Description**: The minimum number of threads used for query.
+
+#### fragment_pool_thread_num_max
+
+- **Default**: 4096
+- **Unit**: N/A
+- **Description**: The maximum number of threads used for query.
+
+#### fragment_pool_queue_size
+
+- **Default**: 2048
+- **Unit**: N/A
+- **Description**: The upper limit of the query number that can be processed on each BE node.
+
+#### enable_token_check
+
+- **Default**: TRUE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to enable the token check. TRUE indicates enabling the token check, and FALSE indicates disabling it.
+
+#### enable_prefetch
+
+- **Default**: TRUE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to enable the pre-fetch of the query. TRUE indicates enabling pre-fetch, and FALSE indicates disabling it.
```

Comment:
Remove "A boolean value to control", and make the sentence start with "Whether".